detector/detector.go

```diff
@@ -419,10 +419,11 @@ func DetectCpeURIsCves(r *models.ScanResult, cpeURIs []string, cnf config.GoCveD
 		if err != nil {
 			return err
 		}
-		for _, detail := range details {
 
+		for _, detail := range details {
 			confidence := models.CpeVersionMatch
 			if detail.IsJvn() {
+				// In the case of CpeVendorProduct-match, only the JVN is set(Nvd is not set).
```

Comment:
If you want to implement it as per this comment, the following expression would be better.
`!details.IsNvd() && detail.IsJvn()`.

Reply:
Certainly. I'll fix it.